open-machine-learning-jupyter-book/data-science/data-visualization/visualizing-quantities.md

```diff
@@ -42,7 +42,7 @@ There are very good 'cheat sheets' available for Matplotlib [here](https://matpl
 Open the file [let-us-learn-about-birds.ipynb](../../assignments/data-science/let-us-learn-about-birds.ipynb) with Jupyter Notebook and add a cell.
 
 ```{note}
-The data is stored in the `/assets/data/` folder.
+The data is stored in the [/assets/data/birds.csv](https://open-academy.github.io/machine-learning/assets/data/birds.csv) file.
```

Comment:
This is the right change https://github.com/open-academy/machine-learning/commit/8dfc3e8a8a222e56fe9f4ed90f8aa80b24fe0dab